numpyro/distributions/continuous.py

```diff
@@ -953,9 +952,9 @@ def variance(self):
 
 @copy_docs_from(Distribution)
 class Pareto(TransformedDistribution):
-    arg_constraints = {'alpha': constraints.positive, 'scale': constraints.positive}
+    arg_constraints = {'scale': constraints.positive, 'alpha': constraints.positive}
 
-    def __init__(self, alpha, scale=1., validate_args=None):
+    def __init__(self, scale, alpha, validate_args=None):
```

Comment:
Pareto args are reordered.

Reply:
One improvement I've been starting to make in Pyro is to clearly separate args from kwargs via `, *,`. This makes it safer to make future changes to kwargs, since kwargs after `, *,` can be specified only by name, not by position. We can't do this pervasively in Pyro because Pyro 1.0 supported Python 2. However since NumPyro is not yet 1.0 you might consider pervasively separating args from kwargs. E.g. you could require `validate_args` to be specified by name
```py
def __init__(self, scale, alpha, *, validate_args=None):
    ...
```
Just a thought 🙂

Reply:
I think this is a good idea. Let me create a separate issue to track down the progress for it.

Reply:
Thanks for your input!